src/index.js

```diff
@@ -39,14 +43,14 @@ app.use(async (req, res, next) => {
 });
 
 // Helper endpoints
-app.get('/', (req, res) => {
-	res.redirect('/api-docs');
+app.get('/', apiLimiter, (req, res) => {
```

Comment:
If you're using the rate limiter everywhere, I think you can do `app.use(apiLimiter)` on its own?

Reply:
The `apiLimiter` is set to one thing for everything, and something else for login. As this matures different endpoints may require different rules depending on public/private/API-access rules we may want to work out.

Reply:
That said I will look into how middleware is overwritten. That is probably a cleaner solution.